sag_ji4.par

```diff
@@ -1,306 +1,306 @@
-! Olympian Sagittal JI notation. Dave Keenan & George Secor. 2007-Nov-11
-! The typical step is ln(3^7/2^11)/ln(2)*1200/233 = 1.960086311 cents
+! Olympian Sagittal JI notation. Dave Keenan & George Secor. 2007-Nov-11, updated 2020-Apr-14
+! The typical step is ln(3^7/2^11)/ln(2)*1200/233 = 0.487918481 cents

```

Comment:
Updated per the version of this file hosted on the main site.

Reply:
Good.